### ISSUE TITLE
Prediction issue

### ISSUE THREAD

**Issue by tim6220** [fastnlp/JointCwsParser#4]
Any API for using the trained model for prediction? Thank you.

**Comment by yhcc** [comment]
No. We don't provide prediction API for this model. But similar methods have been integrated into this package https://github.com/fastnlp/fastHan , this package can be directly called to do prediction.

**Comment by KangChou** [comment]
How should I call the prediction API?
@tim6220  @yhcc
